client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -2,6 +2,8 @@ $       = require 'jquery'
 kd      = require 'kd'
 kookies = require 'kookies'
 
+RECATCHA_JS = 'https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoaded&render=explicit'
```

Comment:
typo: `RECAPTCHA_JS`
and we already have it here: https://github.com/koding/koding/blob/master/client/landing/site.landing/coffee/login/AppView.coffee#L18

you can update it there to use it from one place.